### ISSUE TITLE
[pkg/stanza/operator/input/windows] Failing Windows tests

### ISSUE THREAD

**Issue by dmitryax** [open-telemetry/opentelemetry-collector-contrib#17871]
Tests are failing after https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/17859

https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/3964120410/jobs/6792687244

```
=== RUN   TestUnmarshalRaw
    raw_test.go:92: 
        	Error Trace:	D:\a\opentelemetry-collector-contrib\opentelemetry-collector-contrib\pkg\stanza\operator\input\windows\raw_test.go:92
        	Error:      	Received unexpected error:
        	            	open testdata\rawSample.raw: The system cannot find the file specified.
        	Test:       	TestUnmarshalRaw
--- FAIL: TestUnmarshalRaw (0.00s)
```

**Comment by dmitryax** [comment]
cc @atoulme

**Comment by atoulme** [comment]
Taking a look - I contributed this code today.

**Comment by atoulme** [comment]
Unclear how those tests passed in the first place. The fix is trivial, see #17872.
